### PR TITLE
chore(CI): omit poetry for e2e tests

### PR DESCRIPTION
**Pull request by bojiang**
## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Comment by aarnphm** [comment]
The scripts before works as well

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -261,28 +246,23 @@ jobs:
...
 
       - name: Install dependencies
         if: steps.cache-pip.outputs.cache-hit != 'true' || needs.diff.outputs.related == 'true'
-        run: poetry install -E "model-server docs"
+        run: pip install "." pytest pytest-asyncio pandas
 
-      - name: Run tests and generate coverage report
-        run: make tests-general_features
+      - name: Change to project directory
+        run: cd ./tests/e2e/bento_server_general_features
```
Comment (aarnphm) [inline .github/workflows/ci.yml:262]:
```suggestion
        run: |
          cd ./tests/e2e/bento_server_general_features
          pytest -vvv --cov=bentoml --cov-config=.coveragerc --cov-report=xml:"bento-server.xml"
```
```suggestion
        run: cd ./tests/e2e/bento_server_general_features
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -261,28 +246,23 @@ jobs:
...
 
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@v2
-        with:
-          flags: bento-server
-          directory: ./
-          files: ./tests/e2e/bento_server_general_features/general_features.xml
-          verbose: true
+      - name: Run tests
```
Comment (aarnphm) [inline .github/workflows/ci.yml:264]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -261,28 +246,23 @@ jobs:
...
+        run: pytest -s -vvv
```
Comment (aarnphm) [inline .github/workflows/ci.yml:265]:
```suggestion
        run: pytest -s -vvv
```
```suggestion
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -261,28 +246,23 @@ jobs:
...
+        run: pip install "." pytest pytest-asyncio pandas
 
-      - name: Run tests and generate coverage report
-        run: make tests-general_features
+      - name: Change to project directory
+        run: cd ./tests/e2e/bento_server_general_features
 
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@v2
-        with:
```
Comment (aarnphm) [inline .github/workflows/ci.yml:281]: Can you restore this since we are using carryforward flags for codecov to transfer previous reports on main branch

**Comment by bojiang** [comment]
migrate to https://github.com/bentoml/BentoML/pull/2025 (thx to @aarnphm )

**Pull request closed without merging**